app/views/employees/index.html.erb

```diff
@@ -0,0 +1,33 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Employees</h1>
+
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Email</th>
+      <th>Private number</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @employees.each do |employee| %>
+      <tr>
+        <td><%= employee.name %></td>
+        <td><%= employee.email %></td>
+        <td><%= employee.private_number %></td>
+        <td><%= link_to 'Show', employee %></td>
+        <td><%= link_to 'Edit', edit_employee_path(employee) %></td>
+        <td>
+          <%= button_to 'Delete', employee_path(employee), :method => 'delete', :confirm => 'Are you sure?' %>
+       </td>
+      </tr>
+    <% end %>
+  </tbody>
+</table>
```

Comment:
I recommend that you style this part of the code where all the employees are shown.

Reply:
@YilKanda  The styles for all this scaffold elements are going to be in a different issued.